src/lib/Window.svelte

```diff
@@ -42,13 +42,14 @@
     };
 
     onMount(() => {
-        if (initialMessage)
+        if (initialMessage) {
```

Comment:
Good deal! I think it actually is good practice to use `{}` curly brackets for `ifs` that aren't one line

Reply:
Oh yeah. I was debugging the opening and closing of the chat tutor window and was experimenting around with adding some `else if` and `else` statement in there. But I usually but brackets around my `if` just to feel secure that the code else where is not running. I guess it's a personal preference thing.